npl/pom.xml

```diff
@@ -12,10 +12,6 @@
         <relativePath>../parent-pom</relativePath>
     </parent>
 
-    <properties>
-        <platform.version>2024.1.7</platform.version>
```

Comment:
`platform.version` used for npl-maven-plugin versioning:
![image](https://github.com/user-attachments/assets/217bfb51-4bdb-4945-9185-c073715d757d), 
npl-maven-plugin should be set alternatively if removed, currently causing [failed build](https://github.com/NoumenaDigital/npl-integrations/actions/runs/10266361399/job/28404537668?pr=15)
![image](https://github.com/user-attachments/assets/c5923dd2-4dcc-4e65-a9a1-14a49d411c01)